docs/G-Codes.md

```diff
@@ -476,6 +476,20 @@ enabled.
 `SET_FAN_SPEED FAN=config_name SPEED=<speed>` This command sets the
 speed of a fan. "speed" must be between 0.0 and 1.0.
 
+`SET_FAN_SPEED PIN=config_name TEMPLATE=<template_name>
+[<param_x>=<literal>]`: If `TEMPLATE` is specified then it assigns a
+[display_template](Config_Reference.md#display_template) to the given
```

Comment:
Perhaps `display_template` could get a secondary name of just `template`?
It feels very confusing to be connecting an output pin / fan to a 'display_template' when the intent is to not display anything, but to have it all work in the background...

Maybe `dynamic_variable` or `dynamic_value` or something else might be a good secondary name.  Just something that doesn't tie it specifically to a 'display' usage.

Reply:
Thanks for reviewing.

I agree that display_template is a confusing name.  Roughly I'd like to introduce something like `macro_template`, make `render()` always available from Jinja2, and deprecate `display_template`.  That change seems orthogonal though, so didn't want to complicate this PR with a config rename.

-Kevin